crates/pixi_config/Cargo.toml

```
@@ -26,6 +26,7 @@ serde_json = { workspace = true }
 toml_edit = { workspace = true, features = ["serde"] }
 tracing = { workspace = true }
 url = { workspace = true }
+uv-configuration = { workspace = true }
```

```suggestion
uv-configuration = { workspace = true }
```

Could we redefine the types here and make a conversion method in `pixi_uv_conversions` its better to keep uv dependencies out, as this prohibits us publishing the crates.

---

No problem